scripts/west_commands/runners/pyocd.py

```diff
@@ -50,6 +50,10 @@ def __init__(self, cfg, target,
             frequency_args = ['-f', frequency]
         self.frequency_args = frequency_args
 
+        self.tool_opt = ''
```

Comment:
Can you please use the same style as was used for board_args, daparg_args, and frequency_args here? It will avoid copy/pasting below.